test/integration/targets/docker_swarm_service/tasks/tests/mounts.yml

```diff
@@ -523,3 +523,46 @@
   loop_control:
     loop_var: volume_name
   ignore_errors: yes
+
+- name: mounts.source (empty for tmpfs)
```

Comment:
Could you move this above this line: https://github.com/ansible/ansible/blob/4266786e85426dce7b6f57910ca8efac5cec9c9e/test/integration/targets/docker_swarm_service/tasks/tests/mounts.yml#L512

And add this above the test you added:

```
####################################################################
## mounts.source ###################################################
#################################################################### 
````

This will keep the tests structured.

Reply:
The teardown section of the test should not be below `mounts.source`. Please move these lines https://github.com/ansible/ansible/blob/b7ed6141283cbde24e84dadf1e8c26cc804d9007/test/integration/targets/docker_swarm_service/tasks/tests/mounts.yml#L516-L525

Below https://github.com/ansible/ansible/blob/b7ed6141283cbde24e84dadf1e8c26cc804d9007/test/integration/targets/docker_swarm_service/tasks/tests/mounts.yml#L572

Thanks!